src/Resources/app/storefront/src/checkout/checkout.plugin.js

```diff
@@ -77,13 +77,19 @@ export default class CheckoutPlugin extends Plugin {
         this.data = '';
         this.storedPaymentMethodData = {};
         this.formValidator = {};
+        this.currentPaymentMethodInstance = {};
 
-        // use this object to iterate through the paymentMethods response
-        const paymentMethods = this.adyenCheckout.paymentMethodsResponse.paymentMethods;
-        const storedPaymentMethods = this.adyenCheckout.paymentMethodsResponse.storedPaymentMethods;
+        // Render any preselected Adyen payment method
+        if (adyenCheckoutOptions.selectedPaymentMethodPluginId ===
+            adyenCheckoutOptions.adyenPluginId) {
+            const paymentMethodContainer = $(`[data-adyen-payment-method="${adyenCheckoutOptions.selectedPaymentMethodHandler}"]`);
+            this.renderPaymentMethod(paymentMethodContainer);
+        }
```

Comment:
Blik web component contains a `required` property in the HTML, which prevents selection of any other payment method when blik is rendered.
I refactored the checkout component to mount and unmount the payment method instances on paymentMethod change.
Stored payment methods remain the same (mounted on init)